src/methods.jl

```diff
@@ -120,3 +120,54 @@ julia> flat_correct(frame, flat)
 * [`flat_correct!`](@ref)
 """
 flat_correct(frame::AbstractArray, flat_frame::AbstractArray; kwargs...) = flat_correct!(deepcopy(frame), flat_frame; kwargs...)
+
+
+"""
+    trim(frame::AbstractArray, idxs)
+
+Trims the frame to remove the region specified by idxs.
+
+This function trims the array in a manner such that final array should be a rectangle.
```

Comment:
I specifically changed to "rectangular" because "rectangle" is ambiguous. "the final array should be a rectangle"- does that imply the shape or that the literal array has become a rectangle (ie some single valued array).

Reply:
Sorry, I missed it, I will change it as soon as I wake up!